### PR TITLE
Signal-Desktop: update to 5.0.0

### PR DESCRIPTION
**Pull request by ScrelliCopter**
#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

File: srcpkgs/Signal-Desktop/template [hunk 1]
```diff
@@ -1,22 +1,28 @@
...
 maintainer="Julio Galvan <juliogalvan@protonmail.com>"
 license="GPL-3.0-only"
 homepage="https://github.com/signalapp/Signal-Desktop"
 distfiles="https://github.com/signalapp/Signal-Desktop/archive/v${version}.tar.gz"
-checksum=f9f55c0195579b43712bf370b753337432eda4bcf89809d1c8f3deb0b4ee4400
+checksum=4ff58f5ab5f4de6477dd92069d8c2e9a830513acca527b51e0ee1c7cbe7d8627
 nostrip_files="signal-desktop"
 
 pre_build() {
+	git lfs install
```
Comment (ericonr) [inline srcpkgs/Signal-Desktop/template:19]: I don't know if this is actually ok to do here. What files does it touch?

Comment (ScrelliCopter) [reply srcpkgs/Signal-Desktop/template:19]: @ericonr I don't know, but it's what I needed to get any version past what's in the repos to build 🤷

Comment (ericonr) [reply srcpkgs/Signal-Desktop/template:19]: Man page says

```
DESCRIPTION
Perform the following actions to ensure that Git LFS is setup properly:

○
    Set up the clean and smudge filters under the name "lfs" in the global Git config.
○
    Install a pre-push hook to run git-lfs-pre-push(1) for the current repository, if run from inside one. If "core.hooksPath" is configured in any Git configuration (and supported, i.e., the installed Git version is at least 2.9.0), then the pre-push hook will be installed to that directory instead.
```

@void-linux/pkg-committers is this ok or should we change the position of the git config directory or something?

Comment (ericonr) [reply srcpkgs/Signal-Desktop/template:19]: Anyway, IMO this specific command should be done in `pre_configure`

Comment (q66) [reply srcpkgs/Signal-Desktop/template:19]: if it touches anything outside builddir it's not okay

**Comment by kwshi** [comment]
FYI, this does not build successfully on my machine right now w/ the following error log:

http://ix.io/3lEb

**Comment by ScrelliCopter** [comment]
Closing in favour of #31521.

**Pull request closed without merging**